style.css

```diff
@@ -41,26 +41,40 @@ nav {
   display: none;
 }
 
-.hamburger_toggle {
+.hamburger-menu {
   display: flex;
   justify-content: flex-end;
   align-items: center;
   cursor: pointer;
   margin-left: 80%;
   margin-top: 0.75rem;
-  order: 0;
-  flex-grow: 0;
-  width: 2rem;
+  width: 2.5rem;
   height: 1.75rem;
-  background: var(--background-color);
   border: none;
+  padding-right: 0.25rem;
+  background: rgba(255, 255, 255, 0.8);
 }
 
-.hamburger_toggle img {
-  padding-right: 0.25rem;
-  padding-top: 0.2rem;
+.hamburger-menu span {
+  color: #000;
+  font-size: 2rem;
 }
 
+/*
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 99%;
+  opacity: 0;
+  transform: translateY(-300px);
+  background-color: var(--background-color);
+  color: var(--header-mobile-color);
+  transition: 0.3s;
+} */
+
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 😄